src/filters/AusbauFilters/AusbauFilters.js

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import {
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  makeStyles,
+} from '@material-ui/core';
+import AusbauLayer from '../../layers/AusbauLayer';
+import { updateConstructions } from '../../config/layers';
```

Comment:
Is it really good style to import a function from a config file?

Reply:
Not really but I didn't want to break the cluster and haven't a better idea  right now.

Reply:
maybe the cluster should have it's own class layer and then we just import the layer here and do {code}clusterLayer.refresh(){code}
but it needs more thinking. We will see in the future if we have more use case like this.